README_OSE.md

```diff
@@ -128,5 +144,17 @@ the directory as the inventory) use the -i option for ansible-playbook.
 On the master host:
 ```sh
 systemctl restart openshift-sdn-master
-openshift ex router --create=true --credentials=/var/lib/openshift/openshift.local.certificates/openshift-client/.kubeconfig
+openshift ex router --create=true \
+  --credentials=/var/lib/openshift/openshift.local.certificates/openshift-client/.kubeconfig \
+  --images='docker-buildvm-rhose.usersys.redhat.com:5000/openshift3_beta/ose-${component}:${version}'
```

Comment:
doh, thanks! I didn't notice that it pulled from the wrong location when I tested it out